express/code/scripts/widgets/basic-carousel.css

```diff
@@ -0,0 +1,369 @@
+    width: 540px;
+    height: 540px;
+  }
+  .template-list.horizontal-fullwidth.three.basic-carousel .basic-carousel-container .basic-carousel-platform a picture img {
+    width: 170px;
+    height: 95px;
+  }
+  .template-list.horizontal-fullwidth.four-square.basic-carousel .basic-carousel-container .basic-carousel-platform a picture img {
+    width: 123px;
+    height: 123px;
+  }
+  .template-list.horizontal-fullwidth.four-rectangle.basic-carousel .basic-carousel-container .basic-carousel-platform a picture img {
+    width: 123px;
+    height: 218px;
+  }
+  /* basic carousel platform */
+  .template-list.horizontal-fullwidth.one.basic-carousel .basic-carousel-container .basic-carousel-platform,
+  .template-list.horizontal-fullwidth.three.basic-carousel .basic-carousel-container .basic-carousel-platform,
+  .template-list.horizontal-fullwidth.four-square.basic-carousel .basic-carousel-container .basic-carousel-platform {
+    margin: auto;
+  }
+  .template-list.horizontal-fullwidth.one.basic-carousel .basic-carousel-container .basic-carousel-platform {
+    max-height: unset;
+    height: 540px;
+  }
+  .template-list.horizontal-fullwidth.four-square.basic-carousel .basic-carousel-container .basic-carousel-platform {
+    height: 135px
+  }
+  .template-list.horizontal-fullwidth.four-rectangle.basic-carousel .basic-carousel-container .basic-carousel-platform {
+    height: 275px;
+    margin: auto;
+  }
+  .basic-carousel-left-trigger {
+    margin-left: 0;
+  }
+  .basic-carousel-right-trigger {
+    margin-left: 0;
+  }
+  .basic-carousel-container .template.basic-carousel-element {
+    scroll-snap-align: start;
+  }
+}
+
+@media (min-width: 900px) {
+  .basic-carousel-container .basic-carousel-platform::-webkit-scrollbar {
+    display: none;
+  }
+  .basic-carousel-container a.button.basic-carousel-arrow {
+    margin: 0 10px;
+  }
+  .basic-carousel-container.controls-hidden .basic-carousel-platform {
+    scroll-snap-type: x mandatory;
+  }
+  /* basic carousel carousel containers */
+  .template-list-wrapper:has(.template-list.horizontal-fullwidth.three.basic-carousel) {
+    height: 210px;
+  }
+  .template-list-wrapper:has(.template-list.horizontal-fullwidth.four-rectangle.basic-carousel)  {
+    height: 420px;
+  }
+  .template-list-wrapper:has(.template-list.horizontal-fullwidth.four-square.basic-carousel)  {
+    height: 230px;
+  }
+  /* basic carousel template img */
+  .template-list.horizontal-fullwidth.one.basic-carousel .basic-carousel-container .basic-carousel-platform a picture img {
+    width: 895px;
+    height: 895px;
+  }
+  .template-list.horizontal-fullwidth.three.basic-carousel .basic-carousel-container .basic-carousel-platform a picture img {
+    width: 280px;
+    height: 158px;
+  }
+  .template-list.horizontal-fullwidth.four-rectangle.basic-carousel .basic-carousel-container .basic-carousel-platform a picture img {
+    width: 207px;
```

Comment:
There are a few instances here where these numbers could be vars. Please review and simplify with vars so it can be changed easier in the future.